chinglee-iot/FreeRTOS
# Smp test cases

Add SMP Unit Test Cases
Description
-----------
Add SMP Unit Test Cases after separating them from the framework

Test Steps
-----------
Overall coverage rate:
  lines......: 97.1% (2873 of 2958 lines)
  functions..: 100.0% (216 of 216 functions)
  branches...: 92.0% (1699 of 1847 branches)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
